boards/others/neorv32/board.yml

```diff
@@ -0,0 +1,11 @@
+board:
+  name: neorv32
+  vendor: others
+  revision:
+    format: major.minor.patch
+    default: "1.8.6"
+    revisions:
+      - name: "1.0.0"
```

Comment:
What is revision 1.0.0?

Reply:
1.8.6 is currently the only revision of the IP supported.

Reply:
Fixed